docs/schema/codelists.md

```diff
@@ -194,7 +194,8 @@ The country codelist is used to provide the country component of an address, usi
 
 ### Initiation Type
 
-Contracting processes can be formed under a number of different processes. Currently, only 'tender' is supported in this codelist. Future versions of the standard might support other initiation types. The initiation type is used to provide information to consuming applications on the different blocks of data and releases they can expect from a contracting process.
```

Comment:
We didn't remove the sentence for the deprecated `submissionMethod`. That said, I'm fine with removing this paragraph, because a single-value closed codelist is meaningless anyway. At least for `submissionMethod`, a user of OCDS 1.1 data can use that sentence to interpret the publisher's intent.

Reply:
Aha, and I guess "Future versions of the standard might support other ... " isn't true anymore.